includes/include_mongo_connect.php

```diff
@@ -1,22 +1,26 @@
 <?php
+
+	require_once($_SERVER['DOCUMENT_ROOT'] . "/includes/credentials.php");
+
 	try {
 		// connect to Mongo
 		switch ($_SERVER["SERVER_NAME"]) {
 			case "oastats-dev.mit.edu":
-				$m = new Mongo('mongodb://libdb-dev.mit.edu:27017');
+				// $m = new Mongo( $mongo["dev"] );
+				$m = new Mongo( $mongo["prod"] );
```

Comment:
Is your intent to use one value for all of these cases now and just change it to what you want in the included file? If so, you can remove all of these cases and just set it once. If not, you need to update lines 10 and 17 to use different values than line 23.